manifests/kustomize/sample/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ kind: Kustomization
 bases:
   # Or github.com/kubeflow/pipelines/manifests/kustomize/env/gcp?ref=1.0.0
   - ../env/gcp
+  - prometheus
```

Comment:
suggest moving prometheus to `manifests/kustomize/third-party/prometheus` and refer to it from here as
```
- ../third-party/prometheus
```

Because we want the sample here to be minimal (in the sample folder).

Reply:
done